src/layouts/guides.html

```diff
@@ -71,6 +76,17 @@
         }
       }
 
+      .content-footer {
```

Comment:
maybe we can target specifically with the tag name instead of a new class?  I've been doing that with some of the other CEs on this page, so might make the most sense for consistency.

Also, same feedback here re: using Open Props where possible.

Reply:
Right, but I'm guessing why do we need the wrapper at all, if we just want to target the custom element anyway?  I agree that making it `sticky` and doing all the positioning from the parent makes sense, just not sure why we need to wrap at all?

Is it maybe because CEs are `display:inline` by default, and the `<div>` gives you a `display:block`?

Reply:
Ah, gotcha. I used the wrapper as a means of positioning the anchor and giving it the margin. Let me remove it and give it a shot.